Charts: update the "Drill Down" demo description (#1932)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/DrillDown/description.md

> When the data source has several hierarchical levels, you may want to enable users to navigate between them. The movement from the higher to the lower hierarchical level is called *"drilling down"*. The movement backwards is called *"drilling up"*.

Reviewer: "When data source has". I prefer "if" to "when", unless you need to say "when" to indicate that something will definitely happen. "Has" is a weak verb. I would always try to replace with stronger subject-verb pair. 

The second and third sentences are somewhat parallel in structure and would work well in a list. 

I also wanted to highlight that you "can enable the feature", instead of "enable users to navigate". The latter sounds like some manual work (I will need to implement this on my own). But if we say "you can enable drill-down", it implies that this is a built-in functionality. 

```suggestion
If you bind a TreeMap to a data source with a hierarchical structure, you can enable drill-down and drill-up functionality:  

- *"Drill down"* means navigate to a more detailed view. 
- *"Drill up"* means navigate back up a level.
```

> In this demo, when a user clicks a group of tiles in the TreeMap, the component drills down. To implement this technique, follow the steps below:

Reviewer: You refer only to drill-down in this introductory sentence. Then you talk about drilling-up in one of the list items. I think it makes more sense to talk about both ways in the introduction.

And we don't have to talk about "a user" when we refer to the demo. Demos are for the readers of this text, so we can simply say "you".

```suggestion
In this demo, you can  click/tap a group of TreeMap tiles to drill down. Once you go down a level, navigation links appear at the top of the TreeMap. Use then to drill back up. To implement this technique, follow the steps below:
```

> 1. Set the [interactWithGroup](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#interactWithGroup) property to `true` to enable users to click on groups of items.

Reviewer: As I understand, this is more of a cosmetic step - not really the #1 most important thing that you should do. Even if you don't do this, users can still click on groups. The only issue is that the control will highlight individual items when users move the mouse pointer over them. So this property only prevents that and makes the group act as a whole. 

```suggestion
1. Set the [interactWithGroup](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#interactWithGroup) property to `true` to force the control to hot-track entire groups instead of individual items.
```

2. Call the [node](/Documentation/ApiReference/UI_Components/dxTreeMap/Node/).[drillDown](/Documentation/ApiReference/UI_Components/dxTreeMap/Node/Methods/#drillDown) method in the TreeMap [onClick](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#onClick) handler.

> 3. To be able to get back to another level, you need to create a clickable UI component or a link. This demo generates links in the markup in the [onDrill](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#onDrill) handler that also call the [node](/Documentation/ApiReference/UI_Components/dxTreeMap/Node/).[drillDown](/Documentation/ApiReference/UI_Components/dxTreeMap/Node/Methods/#drillDown) method on click. Inspect the code below for the implementation details.

Reviewer: Since we already mentioned "drilling up" in the introductory sentence, this item's text can be a bit shorter. 

```suggestion
3. To enable drill-up functionality, generate links in the [onDrill](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#onDrill) handler. Call the [node](/Documentation/ApiReference/UI_Components/dxTreeMap/Node/).[drillDown](/Documentation/ApiReference/UI_Components/dxTreeMap/Node/Methods/#drillDown) method on link clicks. Inspect the code below for implementation details.
```
